Makefile

```diff
@@ -110,43 +121,26 @@ check-variables:
 		exit 1; \
 	fi
 
-engine-dependencies: check-variables check-sdk-scripts
-	@./fetch-engine.sh || (printf "Unable to continue without engine files\n"; exit 1)
-	@cd $(ENGINE_DIRECTORY) && make dependencies WIN32=$(WIN32)
-
 engine: check-variables check-sdk-scripts
 	@./fetch-engine.sh || (printf "Unable to continue without engine files\n"; exit 1)
-	@cd $(ENGINE_DIRECTORY) && make core WIN32=$(WIN32)
-
-utility: engine-dependencies engine
-	@test -f "$(ENGINE_DIRECTORY)/OpenRA.Utility.exe" || (printf "OpenRA.Utility.exe not found!\n"; exit 1)
```

Comment:
Why did you decide to remove this?

Reply:
It is redundant - the `engine` target either builds the utility or fails, so why check for it again?